www/_data/tools.yml

```diff
@@ -1,9 +1,10 @@
-# -   name: Example Tool
-#     image: cordova.png
-#     url: http://example.com
+# -   name: GapDebug
+#     image: gapdebug-logo100.png
+#     url: https://www.genuitec.com/products/gapdebug/
 #     description: >
-#         A description that can
-#         span multiple lines.
+#         GapDebug is a free tool for debugging and managing hybrid-mobile Android and iOS apps. 
+#         With Drag and drop App Installation, Instant debugging, and full cross-platform support, 
+#         GapDebug changes how you view debugging for good.
```

Comment:
No problem at all! Don't worry, a PR is made precisely so that you can safely change things and nothing will be affected until it's merged. I'll take a look again whenever you have a moment to update it.